src/js/editor/editor.js

```diff
@@ -306,7 +306,9 @@ class Editor {
 
   // @private
   renderRange() {
-    this.cursor.selectRange(this.range);
+    if (!this.range.isBlank) {
+      this.cursor.selectRange(this.range);
```

Comment:
If the range is blank, I would expect the selection to be cleared. Ignoring the set range seems incorrect.